readme.md

```diff
@@ -209,11 +209,12 @@ Integration tests are run under the `it` profile with the Failsafe plugin using
 
 ## Changelog
 
-### 2.1.4 (XX XXX 2021)
+### 3.0.0 (XX XXX 2021)
 - The minimum version of Jenkins changed to be **2.204.4**
 - A number of dependencies upgrades
 - Fix for JENKINS-63071, we are now setting the repository browser also for multibranch projects. Existing projects will
   need to be opened and saved again to fix this issue. New projects will work without any additional steps.
+- JENKINS-60342 added support for Pull Request triggers
```

Comment:
Are there any plans of releasing this version anytime soon?

Reply:
You are right, we will add more to the change log before release. Thank you for pointing this out to us